independent-publisher-connectors/Strava/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "76586",
+          "scopes": ["activity:read"],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://www.strava.com/api/v3/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://www.strava.com/api/v3/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://www.strava.com/api/v3/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#fc4c02",
```

Comment:
@RichardWie Please refer the [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json) for apiProperties.json of IP Connectors. The default and only one iconBrandColor for all the IP Connectors is `"da3b01"`.